source/_docs/automation.markdown

```diff
@@ -54,3 +54,8 @@ automation:
   trigger:
   ...
 ```
+### Delete Automations
+
+When autionmations remain visible in the Home Assistant Dashboard, even after having deleted in the YAML file, you have to delete them in the UI.
+To delete them completely, go to UI/Developer Tools and find the automation you want to delete under 'Current entities'.
+From there, select the circled i and in the new window at the bottom select 'REMOVE ENTITY'.
```

Comment:
```suggestion

### Delete Automations

When automation remains visible in the Home Assistant Dashboard, even after having deleted in the YAML file, you have to delete them in the UI.

To delete them completely, go to UI/Developer Tools and find the automation you want to delete under 'Current entities'.

From there, select the circled `i` and in the new window at the bottom select 'REMOVE ENTITY'.
```

Reply:
I have no clue why this happens but it happened to me after some other problems.
After searching the forum, I found similar questions but no answers.
Your improvements are fine!

Your right about the route I chose, that's what was given to me as a solution so I thought it was 'the' way to go.
Shall I change it?

Reply:
yes, please